src/item_factory.cpp

```diff
@@ -2977,8 +2988,8 @@ void itype_variant_data::load( const JsonObject &jo )
 {
     alt_name.make_plural();
     mandatory( jo, false, "id", id );
-    mandatory( jo, false, "name", alt_name );
-    mandatory( jo, false, "description", alt_description );
+    optional( jo, false, "name", alt_name, translation() );
+    optional( jo, false, "description", alt_description, translation() );
```

Comment:
This is the default behavior of optional().
```suggestion
    optional( jo, false, "name", alt_name );
    optional( jo, false, "description", alt_description );
```